source/frontend/src/features/leases/detail/LeaseContainer.tsx

```diff
@@ -40,6 +41,7 @@ export interface ILeasePage {
   header?: ReactNode;
   description?: string;
   validation?: Yup.ObjectSchema<any>;
+  claims?: string[] | string;
```

Comment:
string [] or string? if you had just one entry wouldn't [claim] make sense?

Reply:
I agree, I just made it match the router claims type